pkg/statistics/handle/ddl/ddl.go

```diff
@@ -125,9 +130,12 @@ func (h *ddlHandlerImpl) HandleDDLEvent(t *util.DDLEvent) error {
 			// Get the count and modify count of the partition.
 			stats := h.statsHandler.GetPartitionStats(globalTableInfo, def.ID)
 			if stats.Pseudo {
+				is := h.ctx.GetDomainInfoSchema().(infoschema.InfoSchema)
```

Comment:
Usually, this won't happen, so putting the getting scheme operation inside the if statement is OK.